### PR TITLE
init commit for tocoman integration

### PR DESCRIPTION
**Pull request by kaeline**
https://github.com/Metatavu/lipsanen-project-management-spec/issues/3

Endpoints for importing/exporting projects, Project endpoints

### CODE REVIEW

File: swagger.yaml [hunk 1]
```diff
@@ -41,4 +181,50 @@ components:
...
+          $ref: '#/components/schemas/Metadata'
+
+    Task:
+      type: object
+      description: Task object
+      properties:
+        tocomanId:
+          type: integer
+        name:
+          type: string
```
Comment (Daniilmass) [inline swagger.yaml]: This is not used anywhere, should it be in this PR?

File: swagger.yaml [hunk 2]
```diff
@@ -21,6 +21,146 @@ paths:
...
+            schema:
+              type: string
+              format: binary
+      responses:
+        "200":
+          description: List of projects
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/Project'
```
Comment (Daniilmass) [inline swagger.yaml]: When we Import the project - do we need to have a list returned? Just curious

**Review by Daniilmass — CHANGES_REQUESTED** [review]

File: swagger.yaml [hunk 3]
```diff
@@ -21,6 +21,146 @@ paths:
...
+          schema:
+            type: string
+            format: uuid
+      responses:
+        "204":
+          description: No content
+  /v1/projects/{projectId}/export:
+    get:
+      operationId: exportProject
+      summary: Exports project to tocoman xml file
```
Comment (Daniilmass) [inline swagger.yaml]:
```suggestion
      summary: Exports project to Tocoman xml file
```

**Review by Daniilmass — CHANGES_REQUESTED** [review]

File: swagger.yaml (hunk 2 above)
Comment (kaeline) [reply swagger.yaml]: No, it's a result of failed copypaste :)

File: swagger.yaml (hunk 1 above)
Comment (kaeline) [reply swagger.yaml:209]: Tocoman id? Hmm probably not, when making it i assumed it's going to be useful later, but probably not

Comment (kaeline) [reply swagger.yaml]: removed

File: swagger.yaml (hunk 2 above)
Comment (kaeline) [reply swagger.yaml]: Fixed

**Review by Daniilmass — APPROVED** [review]

**Pull request merged**